vapi/simulator/simulator.go

```diff
@@ -291,10 +291,10 @@ func (s *handler) DetachTag(id vim.ManagedObjectReference, tag vim.VslmTagEntry)
 	return nil
 }
 
-// Status responds with custom HTTP status and json encoded val if given.
+// StatusOK responds with http.StatusOK and json encoded val if given.
```

Comment:
```suggestion
// StatusOK responds with http.StatusOK and encodes val, if specified, to JSON
```

Reply:
I found the description to be confusing as I assumed val has to be JSON encoded (which is not true). You might want to clean up the other methods too.